packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1095,11 +1093,13 @@ new PrismaClient({
 
         if (globalThis.NOT_PRISMA_DATA_PROXY) {
```

Comment:
Unrelated: This might deserve a comment why special handling for this case (and what that case really is)